### ISSUE TITLE
Add CharSequence support [Question]

### ISSUE THREAD

**Issue by stepango** [nsk-mironov/smuggler#18]
Is it possible to add CharSequence support?

**Comment by nsk-mironov** [comment]
Yes, it's possible to do and I'll probably add `CharSequence` support in the near future. For now, you can just define the following `TypeAdapter`:

```kotlin
@GlobalAdapter
object CharSequenceTypeAdapter : TypeAdapter<CharSequence> {
  override fun fromParcel(parcel: Parcel): CharSequence {
    return TextUtils.CHAR_SEQUENCE_CREATOR.createFromParcel(parcel)
  }

  override fun toParcel(value: CharSequence, parcel: Parcel, flags: Int) {
    TextUtils.writeToParcel(value, parcel, flags)
  }
}
```

This solution will work in some case, but you may lose some very important information about

**Comment by stepango** [comment]
Thanks a lot for fast response.

**Comment by nsk-mironov** [comment]
Fixed in [0.12.7](https://github.com/nsk-mironov/smuggler/releases/tag/0.12.7)
